contributing-guides/maintainers-guide.md

```diff
@@ -56,17 +56,18 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**,
-  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
+  (2) have all the **review comments addressed and resolved**,
+  (3) get **approved reviews by at least two maintainers**, (the last maintainer can merge immediately after approving) and
   (4) have been open for at least **24 hours** unless the changes are trivial
 
-- If a PR fails to get a review from a second maintainer after a few days,
+- If a PR fails to get a review from a second maintainer after a week,
   the first maintainer should ping others for review. If it still lingers around
   for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
+  the first maintainer can go ahead and merge it while dismissing all reviewers who
+  didn't reviewed the PR.
 
 - If the only issues holding up a merge are **trivial fixes**
-  (typos, syntax errors, etc.), and the author doesn't respond in a day or two,
+  (typos, syntax errors), and the author doesn't respond in a day or two,
```

Comment:
```suggestion
  (typos, syntax errors, etc), and the author doesn't respond in a day or two,
```

We should keep this since its not just typos and errors which are trivial, for example, adding a --help command to a page is trivial.

Reply:
All possible trivial changes should be explicitly specified not to confuse maintainers when to use this rule.

Reply:
Yes, the word trivial can be interpreted differently of course, however, we could include other things into trivial, one suggestion being grammatical errors

Reply:
While being specific is good, but issues holding up a pull request or issue can be many different things that we can't list here exhaustively. For example, not all pull requests are for pages - some PRs are to the client spec, for example, which is often cross-checked by client developers.

Reply:
I've clearly described what changes can be considered as trivial above. ;)